pages/templates/base.html

```diff
@@ -44,9 +44,9 @@
     <div class='collapse navbar-collapse' id='navbarSupportedContent'>
       <ul class='navbar-nav mr-auto'>
         {% if request.user.is_anonymous %}
-        <li class='nav-item'>
-          <a class='nav-link' href='{% url 'signup' %}'>Luo käyttäjä</a>
-        </li>
+        <!-- <li class='nav-item'>
+          <a class='nav-link' href='{# url 'signup'#}'>Luo käyttäjä</a>
```

Comment:
These messed up badly. For some reason HTML comment did not comment out the python tag `{% url 'signup' %}`. So the % signs had to be changed to # signs.